packages/react/src/radio-group/RadioGroup.test.tsx

```diff
@@ -190,15 +222,15 @@ describe('<RadioGroup />', () => {
 
     submitButton.click();
 
-    expect(stringifiedFormData).to.equal('group=');
+    expect(formData!.get('group')).to.equal(null);
```

Comment:
```suggestion
    expect(formData.get('group')).to.equal(null);
```
I think the `!` shouldn't be necessary

Reply:
Sadly no, becuase Formdata is declared like this `let formData: FormData;` so no initialization is happening which will cause TypeScript into thinking this might still be unassigned. 

<img width="829" height="325" alt="Bildschirmfoto 2025-08-10 um 10 09 25" src="https://github.com/user-attachments/assets/116daade-54d5-4137-939e-a86c89f50670" />

Even though we know that onSubmit will run before calling formData.get(...), the compiler can’t guarantee that.

Reply:
Ah, in similar tests I actually put the `expect` inside the submit handler to get around this, e.g. https://github.com/mui/base-ui/blob/master/packages/react/src/slider/root/SliderRoot.test.tsx#L1851-L1857

(because we don't really use the non-null `!`)